package.json

```diff
@@ -48,7 +48,8 @@
       "node-gyp": "^10.0.0",
       "ansi-html": "^0.0.9",
       "fireworm>async": "^2.6.4",
-      "ember-cli-babel>@babel/runtime": "7.27.6"
+      "ember-cli-babel>@babel/runtime": "7.27.6",
+      "package-json>got": "^14.4.7" 
```

Comment:
Should we have just overriden [package-json](https://github.com/sindresorhus/package-json/releases/tag/v10.0.0) to 10+ instead since it looks like they stop using `got`?

Reply:
good point, upgrading package-json is cleaner solution.
also, i am attempting to run `ember try` in the addons that use it and i am getting an error (same error on `main`) im going to look into that.

Reply:
description has been updated w/ how to test `ember try`

Reply:
What's the expectation for `ember try`? I'll admit I never ran it before so it's likely it was already failing but I get failures for `EMBROIDER_TEST_SETUP_OPTIONS` with `safe` and `optimized`

Reply:
Yeah im not sure either. This was also the first time I ran ember try. I am planning to look more into the cases that failed (i should create a ticket for it).